deploy/backplane/mtsre/ocs-converged/10-mtsre-ocs-converged-admins.SubjectPermission.yml

```diff
@@ -0,0 +1,12 @@
+apiVersion: managed.openshift.io/v1alpha1
+kind: SubjectPermission
+metadata:
+  name: backplane-mtsre-admins
```

Comment:
Can you include the addon id in the name? I believe this name has to be unique.

Reply:
Done